src/main/resources/templates/db_scripts/schema.json

```diff
@@ -43,6 +43,19 @@
       "withMetadata": true,
       "populateJsonWithId": true
     },
+    {
+      "tableName": "interface_credentials",
+      "generateId": true,
```

Comment:
In my opinion, there is no need to add `generateId` entry, because PgUtils take care of it and from the next version of RMB it will not be supported.